### ISSUE TITLE
HTTP 403 for images from services.digitalglobe.com

### ISSUE THREAD

**Issue by cschotte** [openstreetmap/iD#9947]
### URL

https://services.digitalglobe.com/earthservice/tmsaccess/tms/1.0.0/DigitalGlobe:ImageryTileService@EPSG:3857@jpg/18/155211/132645.jpg?connectId=ca613e76-811f-46e7-9e1d-84f6795441c2

### How to reproduce the issue?

Open project https://tasks.hotosm.org/projects/11966 and start mapping. You will see a black map without any images. HTTP 403 error for all images from services.digitalglobe.com 

### Which deployed environments do you see the issue in?

Released version at openstreetmap.org/edit

### What version numbers does this issue effect?

2.2.1.1

### Which browsers are you seeing this problem on?

Microsoft Edge

**Comment by navyn13** [comment]
Can you please show screenshot of the issue

**Comment by arch0345** [comment]
Unfortunately we no longer have access to Maxar imagery, see https://github.com/openstreetmap/iD/issues/9710

**Comment by nickrsan** [comment]
Yes, you'll want to contact the manager of that HOT project to have them update the imagery source in use, unfortunately. They've configured a custom imagery source, so it's not an issue with iD, but with the task configuration. I submitted a report of bad imagery on a task within the broader task there already, but if you have any relation with them, you might communicate that directly.

Closing as duplicate.
